privacy_evaluator/output/user_output_privacy_score.py

```diff
@@ -1,38 +1,43 @@
+from matplotlib import pyplot as plt
 import numpy as np
 import pandas as pd
-from matplotlib import pyplot as plt
-
 from typing import Tuple
 
 from .user_output import UserOutput
 
 
 class UserOutputPrivacyScore(UserOutput):
-    """User Output Class
+    """`UserOutputPrivacyScore` class
+
+    Contains the result of a `MembershipInferencePointAnalysis`.
 
     Interpretation of Outcome:
 
     Vulnerability of individual data points:
+    The privacy risk score is an individual sample’s likelihood of being a training member, which allows an adversary to
+    identify samples with high privacy risks and perform membership inference attacks with high confidence [1].
+
     The training data points that exhibit an increased membership privacy risk might differ from their classes mean
     samples (outliers) [2]. You could check them again, see if they have the correct label, or if they exhibit any
     non-standard properties for the class. If so, correct them. It was also shown that points with an high influence on
-    the decision boundary are more vulnerable to membership inference attacks [1]. Therefore, these points might be
+    the decision boundary are more vulnerable to membership inference attacks [3]. Therefore, these points might be
     important. If you want to protect them, you might add several similar training samples as they are to the class.
 
     References:
-    [1] Stacey Truex, Ling Liu, Mehmet Emre Gursoy, Lei Yu, and Wenqi Wei. 2019.Demystifying Membership Inference
-    Attacks in Machine Learning as a Service.IEEE Transactions on Services Computing(2019)
+    [1] Song, Liwei and Prateek Mittal. “Systematic Evaluation of Privacy Risks of Machine Learning Models.” ArXiv
```

Comment:
again, inside a -.py class, I would rather see links to the papers than that reference style.